### PR TITLE
Use ConsoleLogger instead of native Gradle logger

### LINKED ISSUE #2175: Do not make use of native Gradle and Maven loggers
Using the native Gradle logger was probably why [our deprecation messages was overwritten](https://github.com/GoogleContainerTools/jib/pull/2109#issuecomment-555174722).

We should be careful not to make use of the native Maven or Gradle loggers. We can prevent misuse in `ProjectProperties` by just using `consoleLogger`. But it is very easy to accidentally use the native loggers in Maven mojos and Gradle tasks. Unfortunately, now I realized we had used the native loggers when printing deprecation notices.

### PR DESCRIPTION
**Pull request by chanseokoh**
See #2175.

However, I can never figure out why the test doesn't pass on the Travis Java 11 build. Doesn't make sense to me. Still looking into it.

### CODE REVIEW

**Comment by chanseokoh** [comment]
Ah, probably due to concurrency with `ConsoleLogger`.

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java [hunk 1]
```diff
@@ -222,6 +222,10 @@ private static Path zipUpDirectory(Path sourceRoot, Path targetZip) throws IOExc
 
   @Before
   public void setUp() throws IOException, URISyntaxException {
+    Mockito.when(mockLog.isDebugEnabled()).thenReturn(true);
+    Mockito.when(mockLog.isWarnEnabled()).thenReturn(true);
+    Mockito.when(mockLog.isErrorEnabled()).thenReturn(true);
```
Comment (chanseokoh) [inline jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java]: `isInfoEnabled()` is an unnecessary stubbing in Maven.

File: jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java [hunk 2]
```diff
@@ -374,6 +379,7 @@ public void testCreateContainerBuilder_noClassesFiles() throws InvalidImageRefer
         .thenReturn(new TestFileCollection(ImmutableSet.of(nonexistentFile)));
     gradleProjectProperties.createJibContainerBuilder(
         JavaContainerBuilder.from(RegistryImage.named("base")), ContainerizingMode.EXPLODED);
+    gradleProjectProperties.waitForLoggingThread();
```
Comment (TadCordle) [inline jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java]: I noticed we never call this in `MavenProjectPropertiesTest`, should we be?

Comment (chanseokoh) [reply jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java]: `MavenProjectPropertiesTest` doesn't have any `verify(mockLog)`, so need there (yet).

**Pull request merged**